samples/java-customer-registry-kafka-quickstart/README.md

```diff
@@ -27,7 +27,7 @@ mvn compile
 To run the example locally, you must run the Kalix proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
 
 It also contains the configuration to start a local Kafka broker and a Zookeeper that the Kalix proxy will connect to.
-To start the proxy, the Kafka broker, and the Zookeeper, run the following command from the kafka directory:
+To start the proxy, the Kafka broker, and the Zookeeper, run the following command from the `kafka` directory:
```

Comment:
or change the command to `docker-compose -f kafka/docker-compose.yaml up` would make it even more clear/less error prone perhaps?

Reply:
ups, I saw the comment after the approval and merged it directly.